packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -1,19 +1,23 @@
-import React from 'react';
+import React, { useRef, useLayoutEffect, useState } from 'react';
 import styled from 'styled-components';
-import { Icon, variables, IconProps, useTheme } from '@trezor/components';
+import { variables, IconProps, useTheme, Button, Icon, Dropdown } from '@trezor/components';
 import { Translation } from '@suite-components';
 import { useSelector, useActions } from '@suite-hooks';
 import * as routerActions from '@suite-actions/routerActions';
 import { Route } from '@suite-types';
+import useResizeObserver from '@react-hook/resize-observer';
```

Comment:
we already have debounced resize handler in [resizeActions](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/actions/suite/resizeActions.ts) and state stored in [resizeReducer](https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/reducers/suite/resizeReducer.ts)
do we need another dependency for that?

Reply:
I used that, thanks